About page accessed
"GET /about/ HTTP/1.1" 200 13
Index page accessed
"GET / HTTP/1.1" 200 32
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\urls.py changed, reloading.
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 182
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 2764
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 271
Not Found: /docs
"GET /docs HTTP/1.1" 404 2743
"GET /week HTTP/1.1" 301 0
Internal Server Error: /week/
Traceback (most recent call last):
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: week() missing 1 required positional argument: 'client_id'
"GET /week/ HTTP/1.1" 500 62007
C:\Users\Александр\main_fold\djang\dj\myapp\urls.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\urls.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 271
Not Found: /docs
"GET /docs HTTP/1.1" 404 2764
Not Found: /week/
"GET /week/ HTTP/1.1" 404 2767
Internal Server Error: /week/1
Traceback (most recent call last):
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\dj\myapp\views.py", line 11, in week
    client = get_object_or_404(Client, pk=client_id)
                               ^^^^^^
NameError: name 'Client' is not defined
"GET /week/1 HTTP/1.1" 500 65641
Internal Server Error: /week/2
Traceback (most recent call last):
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\dj\myapp\views.py", line 11, in week
    client = get_object_or_404(Client, pk=client_id)
                               ^^^^^^
NameError: name 'Client' is not defined
"GET /week/2 HTTP/1.1" 500 65641
Internal Server Error: /week/0
Traceback (most recent call last):
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\dj\myapp\views.py", line 11, in week
    client = get_object_or_404(Client, pk=client_id)
                               ^^^^^^
NameError: name 'Client' is not defined
"GET /week/0 HTTP/1.1" 500 65641
Internal Server Error: /week/1
Traceback (most recent call last):
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\dj\myapp\views.py", line 11, in week
    client = get_object_or_404(Client, pk=client_id)
                               ^^^^^^
NameError: name 'Client' is not defined
"GET /week/1 HTTP/1.1" 500 65641
Internal Server Error: /week/2
Traceback (most recent call last):
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\dj\myapp\views.py", line 11, in week
    client = get_object_or_404(Client, pk=client_id)
                               ^^^^^^
NameError: name 'Client' is not defined
"GET /week/2 HTTP/1.1" 500 65641
Internal Server Error: /week/1
Traceback (most recent call last):
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\dj\myapp\views.py", line 11, in week
    client = get_object_or_404(Client, pk=client_id)
                               ^^^^^^
NameError: name 'Client' is not defined
"GET /week/1 HTTP/1.1" 500 65641
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 271
Not Found: /docs
"GET /docs HTTP/1.1" 404 2764
Internal Server Error: /week/1
Traceback (most recent call last):
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\dj\myapp\views.py", line 12, in week
    orders = Order.objects.filter(client=client).oreder_by('id')
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'QuerySet' object has no attribute 'oreder_by'. Did you mean: 'order_by'?
"GET /week/1 HTTP/1.1" 500 66407
Internal Server Error: /week/1
Traceback (most recent call last):
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\dj\myapp\views.py", line 12, in week
    orders = Order.objects.filter(client=client).oreder_by('id')
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'QuerySet' object has no attribute 'oreder_by'. Did you mean: 'order_by'?
"GET /week/1 HTTP/1.1" 500 66133
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 271
Not Found: /docs
"GET /docs HTTP/1.1" 404 2764
"GET /week/1 HTTP/1.1" 200 123
"GET /week/1 HTTP/1.1" 200 123
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 271
Not Found: /docs
"GET /docs HTTP/1.1" 404 2764
Internal Server Error: /week/1
Traceback (most recent call last):
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\dj\myapp\views.py", line 14, in week
    return render(request, 'myapp/week.html', {"client": client, "orders": orders})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\defaulttags.py", line 861, in do_for
    nodelist_loop = parser.parse(
                    ^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\base.py", line 515, in parse
    self.unclosed_block_tag(parse_until)
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\base.py", line 578, in unclosed_block_tag
    raise self.error(token, msg)
django.template.exceptions.TemplateSyntaxError: Unclosed tag on line 10: 'for'. Looking for one of: empty, endfor.
"GET /week/1 HTTP/1.1" 500 145003
Internal Server Error: /week/1
Traceback (most recent call last):
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\dj\myapp\views.py", line 14, in week
    return render(request, 'myapp/week.html', {"client": client, "orders": orders})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\defaulttags.py", line 861, in do_for
    nodelist_loop = parser.parse(
                    ^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\base.py", line 515, in parse
    self.unclosed_block_tag(parse_until)
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\base.py", line 578, in unclosed_block_tag
    raise self.error(token, msg)
django.template.exceptions.TemplateSyntaxError: Unclosed tag on line 10: 'for'. Looking for one of: empty, endfor.
"GET /week/1 HTTP/1.1" 500 144729
"GET /week/1 HTTP/1.1" 200 345
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
"GET /week/1 HTTP/1.1" 200 355
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 271
Not Found: /docs
"GET /docs HTTP/1.1" 404 2764
Internal Server Error: /week/1
Traceback (most recent call last):
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\dj\myapp\views.py", line 14, in week
    product = Product.objects.filter(client=client).order_by('id')
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\db\models\manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\db\models\query.py", line 1476, in filter
    return self._filter_or_exclude(False, args, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\db\models\query.py", line 1494, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\db\models\query.py", line 1501, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\db\models\sql\query.py", line 1613, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\db\models\sql\query.py", line 1645, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\db\models\sql\query.py", line 1495, in build_filter
    lookups, parts, reffed_expression = self.solve_lookup_type(arg, summarize)
                                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\db\models\sql\query.py", line 1307, in solve_lookup_type
    _, field, _, lookup_parts = self.names_to_path(lookup_splitted, self.get_meta())
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\db\models\sql\query.py", line 1772, in names_to_path
    raise FieldError(
django.core.exceptions.FieldError: Cannot resolve keyword 'client' into field. Choices are: additional_date, amount, description, id, order, price, product
"GET /week/1 HTTP/1.1" 500 112106
Internal Server Error: /week/1
Traceback (most recent call last):
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\dj\myapp\views.py", line 14, in week
    product = Product.objects.filter(client=client).order_by('id')
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\db\models\manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\db\models\query.py", line 1476, in filter
    return self._filter_or_exclude(False, args, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\db\models\query.py", line 1494, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\db\models\query.py", line 1501, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\db\models\sql\query.py", line 1613, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\db\models\sql\query.py", line 1645, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\db\models\sql\query.py", line 1495, in build_filter
    lookups, parts, reffed_expression = self.solve_lookup_type(arg, summarize)
                                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\db\models\sql\query.py", line 1307, in solve_lookup_type
    _, field, _, lookup_parts = self.names_to_path(lookup_splitted, self.get_meta())
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\db\models\sql\query.py", line 1772, in names_to_path
    raise FieldError(
django.core.exceptions.FieldError: Cannot resolve keyword 'client' into field. Choices are: additional_date, amount, description, id, order, price, product
"GET /week/1 HTTP/1.1" 500 111832
Internal Server Error: /week/1
Traceback (most recent call last):
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\dj\myapp\views.py", line 14, in week
    product = Product.objects.filter(client=client).order_by('id')
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\db\models\manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\db\models\query.py", line 1476, in filter
    return self._filter_or_exclude(False, args, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\db\models\query.py", line 1494, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\db\models\query.py", line 1501, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\db\models\sql\query.py", line 1613, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\db\models\sql\query.py", line 1645, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\db\models\sql\query.py", line 1495, in build_filter
    lookups, parts, reffed_expression = self.solve_lookup_type(arg, summarize)
                                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\db\models\sql\query.py", line 1307, in solve_lookup_type
    _, field, _, lookup_parts = self.names_to_path(lookup_splitted, self.get_meta())
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\db\models\sql\query.py", line 1772, in names_to_path
    raise FieldError(
django.core.exceptions.FieldError: Cannot resolve keyword 'client' into field. Choices are: additional_date, amount, description, id, order, price, product
"GET /week/1 HTTP/1.1" 500 111969
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 271
Not Found: /docs
"GET /docs HTTP/1.1" 404 2764
Internal Server Error: /week/1
Traceback (most recent call last):
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: week() got an unexpected keyword argument 'client_id'
"GET /week/1 HTTP/1.1" 500 62294
Internal Server Error: /week/1
Traceback (most recent call last):
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: week() got an unexpected keyword argument 'client_id'
"GET /week/1 HTTP/1.1" 500 62020
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 271
Internal Server Error: /week/1
Traceback (most recent call last):
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: week() got an unexpected keyword argument 'client_id'
"GET /week/1 HTTP/1.1" 500 62157
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 271
Not Found: /docs
"GET /docs HTTP/1.1" 404 2764
"GET /week/1 HTTP/1.1" 200 355
"GET /week/1 HTTP/1.1" 200 355
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 271
Not Found: /docs
"GET /docs HTTP/1.1" 404 2764
"GET /week/1 HTTP/1.1" 200 355
"GET /week/1 HTTP/1.1" 200 355
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 271
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 2785
Not Found: /docs
"GET /docs HTTP/1.1" 404 2764
"GET /week/1 HTTP/1.1" 200 337
"GET /week/1 HTTP/1.1" 200 337
"GET /week/1 HTTP/1.1" 200 337
Internal Server Error: /week/1
Traceback (most recent call last):
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\dj\myapp\views.py", line 14, in week
    return render(request, 'myapp/week.html', {"client": client, "orders": orders})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\defaulttags.py", line 861, in do_for
    nodelist_loop = parser.parse(
                    ^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\base.py", line 481, in parse
    raise self.error(token, e)
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\base.py", line 479, in parse
    filter_expression = self.compile_filter(token.contents)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\base.py", line 597, in compile_filter
    return FilterExpression(token, self)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\base.py", line 698, in __init__
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: Could not parse the remainder: '().product' from 'order.product.get().product'
"GET /week/1 HTTP/1.1" 500 157948
Internal Server Error: /week/1
Traceback (most recent call last):
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\dj\myapp\views.py", line 14, in week
    return render(request, 'myapp/week.html', {"client": client, "orders": orders})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\defaulttags.py", line 861, in do_for
    nodelist_loop = parser.parse(
                    ^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\base.py", line 481, in parse
    raise self.error(token, e)
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\base.py", line 479, in parse
    filter_expression = self.compile_filter(token.contents)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\base.py", line 597, in compile_filter
    return FilterExpression(token, self)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\base.py", line 698, in __init__
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: Could not parse the remainder: '().product' from 'order.product.get().product'
"GET /week/1 HTTP/1.1" 500 158050
"GET /week/1 HTTP/1.1" 200 337
"GET /week/1 HTTP/1.1" 200 355
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 271
Not Found: /docs
"GET /docs HTTP/1.1" 404 2764
Internal Server Error: /week/1
Traceback (most recent call last):
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\dj\myapp\views.py", line 15, in week
    return render(request, 'myapp/week.html', {"client": client, "orders": orders, "product": product})
                                                                                              ^^^^^^^
NameError: name 'product' is not defined. Did you mean: 'Product'?
"GET /week/1 HTTP/1.1" 500 66486
Internal Server Error: /week/1
Traceback (most recent call last):
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\dj\myapp\views.py", line 15, in week
    return render(request, 'myapp/week.html', {"client": client, "orders": orders, "product": product})
                                                                                              ^^^^^^^
NameError: name 'product' is not defined. Did you mean: 'Product'?
"GET /week/1 HTTP/1.1" 500 66212
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 271
Not Found: /docs
"GET /docs HTTP/1.1" 404 2764
"GET /week/1 HTTP/1.1" 200 337
"GET /week/1 HTTP/1.1" 200 337
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 271
Not Found: /docs
"GET /docs HTTP/1.1" 404 2764
Internal Server Error: /week/1
Traceback (most recent call last):
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\dj\myapp\views.py", line 15, in week
    products = order.products.all()
               ^^^^^^^^^^^^^^
AttributeError: 'Order' object has no attribute 'products'. Did you mean: 'product'?
"GET /week/1 HTTP/1.1" 500 66680
Internal Server Error: /week/1
Traceback (most recent call last):
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\dj\myapp\views.py", line 15, in week
    products = order.products.all()
               ^^^^^^^^^^^^^^
AttributeError: 'Order' object has no attribute 'products'. Did you mean: 'product'?
"GET /week/1 HTTP/1.1" 500 66406
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /week/1
Traceback (most recent call last):
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\dj\myapp\views.py", line 15, in week
    products = order.products.all()
               ^^^^^^^^^^^^^^
AttributeError: 'Order' object has no attribute 'products'. Did you mean: 'product'?
"GET /week/1 HTTP/1.1" 500 66575
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
"GET /week/1 HTTP/1.1" 200 400
"GET /week/1 HTTP/1.1" 200 400
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
"GET /week/1 HTTP/1.1" 200 438
"GET /week/1 HTTP/1.1" 200 438
"GET /week/1 HTTP/1.1" 200 438
"GET /week/1 HTTP/1.1" 200 438
"GET /week/1 HTTP/1.1" 200 438
"GET /week/1 HTTP/1.1" 200 438
"GET /week/1 HTTP/1.1" 200 438
"GET /week/1 HTTP/1.1" 200 438
"GET /week/1 HTTP/1.1" 200 438
"GET /week/1 HTTP/1.1" 200 438
"GET /week/1 HTTP/1.1" 200 438
"GET /week/1 HTTP/1.1" 200 438
"GET /week/1 HTTP/1.1" 200 355
"GET /week/1 HTTP/1.1" 200 337
"GET /week/1 HTTP/1.1" 200 337
"GET /week/1 HTTP/1.1" 200 337
"GET /week/1 HTTP/1.1" 200 337
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 271
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /week/1
Traceback (most recent call last):
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\dj\myapp\views.py", line 14, in week
    for product in order.product.all():
                   ^^^^^
NameError: name 'order' is not defined. Did you mean: 'Order'?
"GET /week/1 HTTP/1.1" 500 66137
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /week/1
Traceback (most recent call last):
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\dj\myapp\views.py", line 14, in week
    for product in orders.product.all():
                   ^^^^^^^^^^^^^^
AttributeError: 'QuerySet' object has no attribute 'product'
"GET /week/1 HTTP/1.1" 500 66295
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 2785
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /week/1
Traceback (most recent call last):
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\dj\myapp\views.py", line 14, in week
    ord = Order.objects.get(pk=client_id)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\db\models\manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\db\models\query.py", line 649, in get
    raise self.model.DoesNotExist(
myapp.models.Order.DoesNotExist: Order matching query does not exist.
"GET /week/1 HTTP/1.1" 500 75164
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /week/1
Traceback (most recent call last):
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\dj\myapp\views.py", line 14, in week
    return render(request, 'myapp/week.html', {"client": client, "orders": orders})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\defaulttags.py", line 861, in do_for
    nodelist_loop = parser.parse(
                    ^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\defaulttags.py", line 860, in do_for
    sequence = parser.compile_filter(bits[in_index + 1])
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\base.py", line 597, in compile_filter
    return FilterExpression(token, self)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\template\base.py", line 698, in __init__
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: Could not parse the remainder: '()' from 'orders.product.all()'
"GET /week/1 HTTP/1.1" 500 165863
"GET /week/1 HTTP/1.1" 200 337
"GET /week/1 HTTP/1.1" 200 337
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 271
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
Not Found: /docs
"GET /docs HTTP/1.1" 404 2764
"GET /week/1 HTTP/1.1" 200 355
"GET /week/1 HTTP/1.1" 200 355
"GET / HTTP/1.1" 200 271
Not Found: /docs
"GET /docs HTTP/1.1" 404 2764
"GET /week/1 HTTP/1.1" 200 355
"GET /week/1 HTTP/1.1" 200 355
Not Found: /week/0
"GET /week/0 HTTP/1.1" 404 2592
"GET /week/2 HTTP/1.1" 200 355
Not Found: /week/3
"GET /week/3 HTTP/1.1" 404 2592
Not Found: /week/4
"GET /week/4 HTTP/1.1" 404 2592
"GET /week/2 HTTP/1.1" 200 355
"GET /week/2 HTTP/1.1" 200 355
"GET /week/1 HTTP/1.1" 200 355
"GET /week/1 HTTP/1.1" 200 355
"GET /week/2 HTTP/1.1" 200 355
"GET /week/2 HTTP/1.1" 200 355
C:\Users\Александр\main_fold\djang\dj\myapp\models.py changed, reloading.
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 271
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 345
Not Found: /docs
"GET /docs HTTP/1.1" 404 2764
"GET /week/1 HTTP/1.1" 200 355
"GET /week/1 HTTP/1.1" 200 355
"GET / HTTP/1.1" 200 416
"GET / HTTP/1.1" 200 414
"GET / HTTP/1.1" 200 414
"GET / HTTP/1.1" 200 414
"GET / HTTP/1.1" 200 414
"GET / HTTP/1.1" 200 394
"GET / HTTP/1.1" 200 502
"GET /week/1 HTTP/1.1" 200 355
Not Found: /month/1
"GET /month/1 HTTP/1.1" 404 2773
Not Found: /year/1
"GET /year/1 HTTP/1.1" 404 2770
"GET /week/1 HTTP/1.1" 200 355
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 502
Not Found: /year/1
"GET /year/1 HTTP/1.1" 404 2770
C:\Users\Александр\main_fold\djang\dj\myapp\urls.py changed, reloading.
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 502
"GET /month/1 HTTP/1.1" 200 355
"GET /year/1 HTTP/1.1" 200 355
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 2827
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 502
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 2827
Internal Server Error: /week/1
Traceback (most recent call last):
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\dj\myapp\views.py", line 15, in week
    order_date__gte=datetime.now() - timedelta(days=7)
                    ^^^^^^^^
NameError: name 'datetime' is not defined. Did you forget to import 'datetime'
"GET /week/1 HTTP/1.1" 500 65838
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 502
"GET /week/1 HTTP/1.1" 200 355
"GET /week/2 HTTP/1.1" 200 355
"GET /week/2 HTTP/1.1" 200 355
Not Found: /week/3
"GET /week/3 HTTP/1.1" 404 2592
"GET /week/2 HTTP/1.1" 200 355
"GET /week/2 HTTP/1.1" 200 355
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 502
"GET /week/1 HTTP/1.1" 200 355
"GET /week/2 HTTP/1.1" 200 353
"GET /week/2 HTTP/1.1" 200 353
"GET /week/2 HTTP/1.1" 200 353
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
"GET /week/2 HTTP/1.1" 200 353
"GET /month/1 HTTP/1.1" 200 355
"GET /year/1 HTTP/1.1" 200 355
"GET /year/2 HTTP/1.1" 200 353
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 502
Internal Server Error: /month/1
Traceback (most recent call last):
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\dj\myapp\views.py", line 25, in month
    end_date = start_date + relativedelta.relativedelta(days=+monthrange(current_year, current_month)[1])
                            ^^^^^^^^^^^^^
NameError: name 'relativedelta' is not defined
"GET /month/1 HTTP/1.1" 500 66581
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 502
"GET /week/1 HTTP/1.1" 200 355
"GET /week/2 HTTP/1.1" 200 473
"GET /week/2 HTTP/1.1" 200 473
"GET /month/1 HTTP/1.1" 200 355
"GET /month/2 HTTP/1.1" 200 473
"GET /year/1 HTTP/1.1" 200 457
"GET /year/2 HTTP/1.1" 200 677
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 502
"GET /year/1 HTTP/1.1" 200 355
"GET /year/2 HTTP/1.1" 200 353
"GET /month/1 HTTP/1.1" 200 355
"GET /month/2 HTTP/1.1" 200 353
"GET /week/1 HTTP/1.1" 200 355
"GET /week/2 HTTP/1.1" 200 353
"GET /week/2 HTTP/1.1" 200 353
"GET /year/2 HTTP/1.1" 200 353
"GET /week/2 HTTP/1.1" 200 353
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
"GET /week/2 HTTP/1.1" 200 353
"GET /month/1 HTTP/1.1" 200 355
Not Found: /month/
"GET /month/ HTTP/1.1" 404 2812
"GET /month/2 HTTP/1.1" 200 353
"GET /year/1 HTTP/1.1" 200 457
"GET /year/2 HTTP/1.1" 200 659
"GET /year/2 HTTP/1.1" 200 659
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /month/1
Traceback (most recent call last):
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\dj\myapp\views.py", line 25, in month
    end_date = start_date + relativedelta.relativedelta(days=+monthrange(current_year, current_month)[1])
                            ^^^^^^^^^^^^^
NameError: name 'relativedelta' is not defined
"GET /month/1 HTTP/1.1" 500 66581
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 502
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
"GET /month/1 HTTP/1.1" 200 355
"GET /year/1 HTTP/1.1" 200 355
"GET /year/2 HTTP/1.1" 200 353
"GET /year/2 HTTP/1.1" 200 353
"GET /month/1 HTTP/1.1" 200 355
"GET /month/2 HTTP/1.1" 200 353
"GET /month/2 HTTP/1.1" 200 353
"GET /week/1 HTTP/1.1" 200 232
"GET /week/2 HTTP/1.1" 200 353
"GET /week/2 HTTP/1.1" 200 353
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
"GET /week/2 HTTP/1.1" 200 353
"GET /year/1 HTTP/1.1" 200 457
"GET /year/2 HTTP/1.1" 200 659
"GET /year/2 HTTP/1.1" 200 659
"GET /year/2 HTTP/1.1" 200 659
"GET /year/2 HTTP/1.1" 200 659
C:\Users\Александр\main_fold\djang\dj\myapp\models.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\models.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\models.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 502
"GET /year/1 HTTP/1.1" 200 457
"GET /month/1 HTTP/1.1" 200 355
"GET /year/1 HTTP/1.1" 200 457
"GET /year/2 HTTP/1.1" 200 780
"GET /year/2 HTTP/1.1" 200 780
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
"GET /year/2 HTTP/1.1" 200 474
C:\Users\Александр\main_fold\djang\dj\myapp\models.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 502
"GET /week/2 HTTP/1.1" 200 353
"GET /month/2 HTTP/1.1" 200 475
"GET /year/2 HTTP/1.1" 200 596
"GET /month/2 HTTP/1.1" 200 475
"GET /year/2 HTTP/1.1" 200 596
"GET /month/2 HTTP/1.1" 200 475
"GET /week/2 HTTP/1.1" 200 353
"GET /month/2 HTTP/1.1" 200 475
"GET /year/2 HTTP/1.1" 200 596
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 502
"GET /week/2 HTTP/1.1" 200 353
"GET /month/2 HTTP/1.1" 200 475
"GET /year/2 HTTP/1.1" 200 596
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 502
"GET /week/2 HTTP/1.1" 200 353
"GET /week/2 HTTP/1.1" 200 353
"GET /month/2 HTTP/1.1" 200 475
"GET /week/2 HTTP/1.1" 200 353
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 502
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 2827
Internal Server Error: /year/2
Traceback (most recent call last):
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\dj\myapp\views.py", line 38, in year
    order = Order.objects.get(pk=client_id)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\db\models\manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\db\models\query.py", line 649, in get
    raise self.model.DoesNotExist(
myapp.models.Order.DoesNotExist: Order matching query does not exist.
"GET /year/2 HTTP/1.1" 500 75682
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 502
Internal Server Error: /year/2
Traceback (most recent call last):
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\dj\myapp\views.py", line 38, in year
    ord = Order.objects.get(pk=client_id)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\db\models\manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\db\models\query.py", line 649, in get
    raise self.model.DoesNotExist(
myapp.models.Order.DoesNotExist: Order matching query does not exist.
"GET /year/2 HTTP/1.1" 500 75672
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /year/2
Traceback (most recent call last):
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\dj\myapp\views.py", line 38, in year
    ord = Order.objects.get(pk=2)
          ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\db\models\manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\db\models\query.py", line 649, in get
    raise self.model.DoesNotExist(
myapp.models.Order.DoesNotExist: Order matching query does not exist.
"GET /year/2 HTTP/1.1" 500 75777
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /year/2
Traceback (most recent call last):
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\dj\myapp\views.py", line 45, in year
    product = orders.product.all()
              ^^^^^^^^^^^^^^
AttributeError: 'QuerySet' object has no attribute 'product'
"GET /year/2 HTTP/1.1" 500 66396
Internal Server Error: /year/2
Traceback (most recent call last):
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\dj\myapp\views.py", line 45, in year
    product = orders.product.all()
              ^^^^^^^^^^^^^^
AttributeError: 'QuerySet' object has no attribute 'product'
"GET /year/2 HTTP/1.1" 500 66394
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /year/2
Traceback (most recent call last):
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\dj\myapp\views.py", line 45, in year
    product = orders.product_id.all()
              ^^^^^^^^^^^^^^^^^
AttributeError: 'QuerySet' object has no attribute 'product_id'
"GET /year/2 HTTP/1.1" 500 66418
Internal Server Error: /year/2
Traceback (most recent call last):
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\dj\myapp\views.py", line 45, in year
    product = orders.product_id.all()
              ^^^^^^^^^^^^^^^^^
AttributeError: 'QuerySet' object has no attribute 'product_id'
"GET /year/2 HTTP/1.1" 500 66418
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /year/2
Traceback (most recent call last):
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\dj\myapp\views.py", line 45, in year
    product = orders.product_id.all()
              ^^^^^^^^^^^^^^^^^
AttributeError: 'QuerySet' object has no attribute 'product_id'
"GET /year/2 HTTP/1.1" 500 66420
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /year/2
Traceback (most recent call last):
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\dj\myapp\views.py", line 47, in year
    c = c + order
        ~~^~~~~~~
TypeError: unsupported operand type(s) for +: 'NoneType' and 'Order'
"GET /year/2 HTTP/1.1" 500 66800
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /year/2
Traceback (most recent call last):
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Александр\main_fold\djang\dj\myapp\views.py", line 47, in year
    c = c + order
        ~~^~~~~~~
TypeError: can only concatenate str (not "Order") to str
"GET /year/2 HTTP/1.1" 500 66722
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
"GET /year/2 HTTP/1.1" 200 596
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
"GET /year/2 HTTP/1.1" 200 596
"GET /year/2 HTTP/1.1" 200 596
"GET /year/2 HTTP/1.1" 200 596
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 502
"GET /week/2 HTTP/1.1" 200 353
"GET /month/2 HTTP/1.1" 200 475
"GET /year/2 HTTP/1.1" 200 596
C:\Users\Александр\main_fold\djang\dj\myapp\models.py changed, reloading.
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 502
"GET /month/2 HTTP/1.1" 200 475
"GET /year/2 HTTP/1.1" 200 717
"GET /week/2 HTTP/1.1" 200 353
"GET /month/2 HTTP/1.1" 200 475
"GET /year/2 HTTP/1.1" 200 717
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 502
"GET /month/2 HTTP/1.1" 200 594
"GET /year/2 HTTP/1.1" 200 836
"GET /month/2 HTTP/1.1" 200 594
"GET /year/2 HTTP/1.1" 200 836
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 502
"GET /week/2 HTTP/1.1" 200 472
"GET /week/1 HTTP/1.1" 200 232
"GET /week/1 HTTP/1.1" 200 232
"GET /month/2 HTTP/1.1" 200 594
"GET /month/1 HTTP/1.1" 200 355
"GET /year/2 HTTP/1.1" 200 836
"GET /year/1 HTTP/1.1" 200 355
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 502
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 2827
"GET /week/2 HTTP/1.1" 200 232
"GET /month/2 HTTP/1.1" 200 594
"GET /week/2 HTTP/1.1" 200 232
"GET /month/2 HTTP/1.1" 200 594
"GET /year/2 HTTP/1.1" 200 836
"GET /month/2 HTTP/1.1" 200 594
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 502
Not Found: /docs
"GET /docs HTTP/1.1" 404 3003
Not Found: /upload
"GET /upload HTTP/1.1" 404 3009
"GET /upload/1 HTTP/1.1" 200 591
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\views.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 601
Not Found: /image/2
"GET /image/2 HTTP/1.1" 404 3012
Not Found: /image/2
"GET /image/2 HTTP/1.1" 404 3012
Not Found: /image/2
"GET /image/2 HTTP/1.1" 404 3012
Not Found: /image/2
"GET /image/2 HTTP/1.1" 404 3012
"GET / HTTP/1.1" 200 602
"GET /upload/2 HTTP/1.1" 200 591
"POST /upload/2 HTTP/1.1" 200 591
"GET /upload/1 HTTP/1.1" 200 591
"POST /upload/1 HTTP/1.1" 200 591
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 602
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 3024
Not Found: /docs
"GET /docs HTTP/1.1" 404 3003
"GET /admin HTTP/1.1" 301 0
"GET /admin/ HTTP/1.1" 302 0
"GET /admin/login/?next=/admin/ HTTP/1.1" 200 4359
"GET /static/admin/css/base.css HTTP/1.1" 200 21544
"GET /static/admin/css/responsive.css HTTP/1.1" 200 17905
"GET /static/admin/css/nav_sidebar.css HTTP/1.1" 200 2810
"GET /static/admin/css/dark_mode.css HTTP/1.1" 200 2682
"GET /static/admin/css/login.css HTTP/1.1" 200 958
"GET /static/admin/js/theme.js HTTP/1.1" 200 1943
"GET /static/admin/js/nav_sidebar.js HTTP/1.1" 200 3063
"POST /admin/login/?next=/admin/ HTTP/1.1" 302 0
"GET /admin/ HTTP/1.1" 200 7499
"GET /static/admin/css/dashboard.css HTTP/1.1" 200 441
"GET /static/admin/img/icon-changelink.svg HTTP/1.1" 200 380
"GET /static/admin/img/icon-addlink.svg HTTP/1.1" 200 331
"GET /admin/myapp/client/ HTTP/1.1" 200 11807
"GET /static/admin/css/changelists.css HTTP/1.1" 200 6811
"GET /static/admin/js/jquery.init.js HTTP/1.1" 200 347
"GET /static/admin/js/core.js HTTP/1.1" 200 6208
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /static/admin/js/admin/RelatedObjectLookups.js HTTP/1.1" 200 9042
"GET /static/admin/js/urlify.js HTTP/1.1" 200 7887
"GET /static/admin/js/actions.js HTTP/1.1" 200 8067
"GET /static/admin/js/vendor/jquery/jquery.js HTTP/1.1" 200 285314
"GET /static/admin/js/prepopulate.js HTTP/1.1" 200 1531
"GET /static/admin/js/vendor/xregexp/xregexp.js HTTP/1.1" 200 325171
"GET /static/admin/js/filters.js HTTP/1.1" 200 978
"GET /static/admin/img/tooltag-add.svg HTTP/1.1" 200 331
C:\Users\Александр\main_fold\djang\dj\myapp\admin.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\admin.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\admin.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\admin.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\admin.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\admin.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\admin.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\admin.py changed, reloading.
Watching for file changes with StatReloader
"GET /admin/myapp/client/ HTTP/1.1" 200 13317
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
C:\Users\Александр\main_fold\djang\dj\myapp\admin.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\admin.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\admin.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\admin.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\admin.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\admin.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\admin.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\admin.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\admin.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\admin.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\admin.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\admin.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\admin.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\admin.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\admin.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\admin.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\admin.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\admin.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\admin.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\admin.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\admin.py changed, reloading.
Watching for file changes with StatReloader
"GET /admin/myapp/client/ HTTP/1.1" 200 13627
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /static/admin/img/sorting-icons.svg HTTP/1.1" 200 1097
"GET /admin/myapp/order/ HTTP/1.1" 200 18832
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /static/admin/img/icon-viewlink.svg HTTP/1.1" 200 581
"GET /admin/myapp/client/ HTTP/1.1" 200 13627
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /admin/myapp/order/ HTTP/1.1" 200 18832
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /admin/myapp/order/?order_date__gte=2024-04-30+00%3A00%3A00%2B00%3A00&order_date__lt=2024-05-08+00%3A00%3A00%2B00%3A00 HTTP/1.1" 200 16040
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /admin/myapp/order/?order_date__gte=2024-05-01+00%3A00%3A00%2B00%3A00&order_date__lt=2024-06-01+00%3A00%3A00%2B00%3A00 HTTP/1.1" 200 16040
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /admin/myapp/order/?order_date__gte=2024-01-01+00%3A00%3A00%2B00%3A00&order_date__lt=2025-01-01+00%3A00%3A00%2B00%3A00 HTTP/1.1" 200 17499
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /admin/myapp/order/?order_date__gte=2024-05-07+00%3A00%3A00%2B00%3A00&order_date__lt=2024-05-08+00%3A00%3A00%2B00%3A00 HTTP/1.1" 200 12912
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /admin/myapp/order/? HTTP/1.1" 200 18832
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /admin/myapp/order/?order_date__gte=2024-04-30+00%3A00%3A00%2B00%3A00&order_date__lt=2024-05-08+00%3A00%3A00%2B00%3A00 HTTP/1.1" 200 16040
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /admin/myapp/order/?order_date__gte=2024-04-30+00%3A00%3A00%2B00%3A00&order_date__lt=2024-05-08+00%3A00%3A00%2B00%3A00 HTTP/1.1" 200 16040
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /admin/myapp/order/? HTTP/1.1" 200 18832
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"POST /admin/myapp/order/? HTTP/1.1" 200 9803
"GET /static/admin/js/cancel.js HTTP/1.1" 200 884
"POST /admin/myapp/order/? HTTP/1.1" 302 0
"GET /admin/myapp/order/ HTTP/1.1" 200 16850
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /static/admin/img/icon-yes.svg HTTP/1.1" 200 436
"GET /admin/myapp/product/ HTTP/1.1" 200 13742
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /static/admin/img/search.svg HTTP/1.1" 200 458
"GET /admin/myapp/client/ HTTP/1.1" 200 13627
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /admin/myapp/client/7/change/ HTTP/1.1" 200 11672
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /static/admin/css/forms.css HTTP/1.1" 200 9090
"GET /static/admin/js/prepopulate_init.js HTTP/1.1" 200 586
"GET /static/admin/css/widgets.css HTTP/1.1" 200 11800
"GET /static/admin/js/change_form.js HTTP/1.1" 200 606
"GET / HTTP/1.1" 200 602
Not Found: /docs
"GET /docs HTTP/1.1" 404 3003
"GET /admin/ HTTP/1.1" 200 9112
"GET /static/admin/img/icon-deletelink.svg HTTP/1.1" 200 392
"GET /admin/myapp/client/ HTTP/1.1" 200 13627
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /admin/myapp/order/ HTTP/1.1" 200 16705
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /admin/myapp/product/ HTTP/1.1" 200 13742
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /admin/myapp/product/ HTTP/1.1" 200 13742
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /admin/auth/user/ HTTP/1.1" 200 13054
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /admin/myapp/client/ HTTP/1.1" 200 13627
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /admin/myapp/order/ HTTP/1.1" 200 16705
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /admin/myapp/order/ HTTP/1.1" 200 16705
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /admin/myapp/product/ HTTP/1.1" 200 13742
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /admin/myapp/ HTTP/1.1" 200 6274
"GET /admin/myapp/ HTTP/1.1" 200 6274
"GET /admin/myapp/client/ HTTP/1.1" 200 13627
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
C:\Users\Александр\main_fold\djang\dj\myapp\admin.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\admin.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\admin.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\admin.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\admin.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\admin.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\admin.py changed, reloading.
Watching for file changes with StatReloader
"GET /admin/myapp/client/ HTTP/1.1" 200 13627
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /admin/myapp/client/1/change/ HTTP/1.1" 200 12340
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /admin/myapp/order/ HTTP/1.1" 200 16705
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /admin/myapp/order/3/change/ HTTP/1.1" 200 13151
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
C:\Users\Александр\main_fold\djang\dj\myapp\admin.py changed, reloading.
Watching for file changes with StatReloader
"GET /admin/myapp/order/3/change/ HTTP/1.1" 200 12120
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /admin/myapp/client/1/change/ HTTP/1.1" 200 12340
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /admin/myapp/product/ HTTP/1.1" 200 13742
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /admin/myapp/product/1/change/ HTTP/1.1" 200 12472
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
Not Found: /media/5023c2d7ac05d7f9377853d19eef22e3.jpeg
"GET /media/5023c2d7ac05d7f9377853d19eef22e3.jpeg HTTP/1.1" 404 3120
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 602
Not Found: /docs
"GET /docs HTTP/1.1" 404 3003
"GET /admin/ HTTP/1.1" 200 9112
"GET /admin/myapp/product/ HTTP/1.1" 200 13742
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /admin/myapp/order/ HTTP/1.1" 200 17081
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /admin/myapp/client/ HTTP/1.1" 200 13627
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
C:\Users\Александр\main_fold\djang\dj\myapp\admin.py changed, reloading.
Watching for file changes with StatReloader
"GET /admin/myapp/client/ HTTP/1.1" 200 15167
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /admin/myapp/client/?register_date__gte=2024-04-30+00%3A00%3A00%2B00%3A00&register_date__lt=2024-05-08+00%3A00%3A00%2B00%3A00 HTTP/1.1" 200 15459
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /admin/myapp/client/?register_date__gte=2024-05-07+00%3A00%3A00%2B00%3A00&register_date__lt=2024-05-08+00%3A00%3A00%2B00%3A00 HTTP/1.1" 200 11524
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /admin/myapp/client/?register_date__gte=2024-05-01+00%3A00%3A00%2B00%3A00&register_date__lt=2024-06-01+00%3A00%3A00%2B00%3A00 HTTP/1.1" 200 15459
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /admin/myapp/client/?register_date__gte=2024-01-01+00%3A00%3A00%2B00%3A00&register_date__lt=2025-01-01+00%3A00%3A00%2B00%3A00 HTTP/1.1" 200 16928
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /admin/myapp/client/?register_date__gte=2024-05-07+00%3A00%3A00%2B00%3A00&register_date__lt=2024-05-08+00%3A00%3A00%2B00%3A00 HTTP/1.1" 200 11524
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /admin/myapp/client/? HTTP/1.1" 200 15167
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /admin/myapp/client/?_facets=True HTTP/1.1" 200 15689
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /static/admin/img/icon-hidelink.svg HTTP/1.1" 200 784
"GET /admin/myapp/client/? HTTP/1.1" 200 15167
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 602
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 3024
Not Found: /docs
"GET /docs HTTP/1.1" 404 3003
"GET /admin/ HTTP/1.1" 200 9112
"GET /admin/ HTTP/1.1" 200 9112
"GET /admin/myapp/client/ HTTP/1.1" 200 15167
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /admin/myapp/client/1/change/ HTTP/1.1" 200 12585
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /static/admin/js/collapse.js HTTP/1.1" 200 1803
"GET /admin/myapp/order/ HTTP/1.1" 200 17081
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /admin/myapp/order/3/change/ HTTP/1.1" 200 12120
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
C:\Users\Александр\main_fold\djang\dj\myapp\admin.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\admin.py changed, reloading.
Watching for file changes with StatReloader
"GET /admin/myapp/order/3/change/ HTTP/1.1" 200 12120
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /admin/myapp/order/ HTTP/1.1" 200 17081
"GET /admin/myapp/order/3/change/ HTTP/1.1" 200 12120
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
C:\Users\Александр\main_fold\djang\dj\myapp\admin.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\admin.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\admin.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\Александр\main_fold\djang\dj\myapp\admin.py changed, reloading.
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 602
Not Found: /docs
"GET /docs HTTP/1.1" 404 3003
"GET /admin/ HTTP/1.1" 200 9112
"GET /admin/ HTTP/1.1" 200 9112
"GET /admin/myapp/client/ HTTP/1.1" 200 15167
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /admin/myapp/client/1/change/ HTTP/1.1" 200 12457
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /admin/myapp/order/ HTTP/1.1" 200 17081
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /admin/myapp/order/3/change/ HTTP/1.1" 200 12235
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /admin/myapp/product/ HTTP/1.1" 200 13742
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /admin/myapp/product/1/change/ HTTP/1.1" 200 13895
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /admin/myapp/product/ HTTP/1.1" 200 13742
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /admin/myapp/product/2/change/ HTTP/1.1" 200 13835
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
"GET /admin/myapp/client/ HTTP/1.1" 200 15167
"GET /admin/jsi18n/ HTTP/1.1" 200 17546
